runtime/gc_structs/ClassIteratorClassSlots.hpp

```diff
@@ -62,6 +62,7 @@ enum {
 class GC_ClassIteratorClassSlots
 {
 protected:
+	bool _shouldScanInterfaces;
```

Comment:
Sorry I did not mention this first time. I think it might be even `const bool  _shouldScanInterfaces;` to improve compiler's ability to optimize code better.